spec/recite_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'recite'
+
+describe Recite do
+	describe "#recite" do
+		it "returns the nursery rhyme" do
+			expect(subject.recite).to eq("This is the house that Jack built.\n"\
+			"This is the malt that lay in the house that Jack built.\n"\
+			"This is the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"\
+			"This is the horse and the hound and the horn that belonged to the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built."
+			)
+		end
+	end
+end
```

Comment:
issue: We typically use 2 spaces for identation. Please convert.

Reply:
Push space indentation.